pynamodb/connection/base.py

```diff
@@ -189,16 +191,15 @@ def __init__(self, region=None, host=None, session_cls=None):
         else:
             self.region = DEFAULT_REGION
 
-        # TODO: provide configurability of retry parameters via arguments
-        self._request_timeout_seconds = DEFAULT_TIMEOUT
-        self._max_retry_attempts_exception = DEFAULT_MAX_RETRY_ATTEMPTS_EXCEPTION
-        self._base_backoff_ms = DEFAULT_BASE_BACKOFF_MS
-
         if session_cls:
             self.session_cls = session_cls
         else:
             self.session_cls = requests.Session
 
+        self._request_timeout_seconds = get_settings_value('REQUEST_TIMEOUT_SECONDS') if request_timeout_seconds is None else request_timeout_seconds
```

Comment:
let's write them all out longform instead of shortened. lines start getting long